physics/module_sf_noahmplsm.F90

```diff
@@ -4965,7 +5084,7 @@ subroutine sfcdif1(parameters,iter   ,sfctmp ,rhoair ,h      ,qair   , & !in
 
     real (kind=kind_phys),                intent(out) :: cm     !< drag coefficient for momentum
     real (kind=kind_phys),                intent(out) :: ch     !< drag coefficient for heat
-    real (kind=kind_phys),                intent(out) :: fv     !< friction velocity (m/s)
+    real (kind=kind_phys),              intent(inout) :: fv     !< friction velocity (m/s)
```

Comment:
Same comment about `fv` here. Please make sure that comments are not referring to it as intent(out) only anymore.